### DISCUSSION TITLE
Feature: Markdownバックアップ用のAPIがほしい

### DISCUSSION THREAD

**Discussion by AnaTofuZ** [growilabs/growi#7713] [Ideas / Requests]
Growiを利用していて、markdownそのものをVCSで管理したいと思うときがあります。
またそれをAPI経由で行いたいと考えています。
現状はpages.listのAPIにアクセスし、それぞれのpageの内容のjsonを取得する方針で行っていますが、この場合publicに公開されている記事のみしかAPI経由で取得できないため、チーム限定や個人限定のエントリがdonwloadできないです。

管理者権限があるユーザーのトークンでapiを叩いたときのみ、すべてのエントリにアクセスするなどの機能があれば嬉しいと思い、提案してみました

**Comment by yuki-takei** [comment]
@AnaTofuZ リクエストありがとうございます。検討します。

## ロールについて

現在のロールが管理者と一般ユーザーしかないのも課題ではあると思っていて、まずそこに手を入れてからの方がいいかもしれないですね。Only me にしてあるページのデータを、そこそこの数存在する管理者ユーザーが取得出来るという仕様変更は、既に導入済みの組織にとって問題になるかもしれないので…

## 優先順について

現在の開発チームは、bootstrap4 化およびサイドバーの実装を優先して動いているので、その辺りが一段落してから棚卸しということになると思います。お待たせすることになると思いますがよろしくお願いいたします。
